package.json

```diff
@@ -6,10 +6,11 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "preview": "vite preview",
```

Comment:
May ask you why `preview` script was removed? 🤔

Reply:
My apologies, I think I have overwritten somehow while adding scripts for test

Reply:
Fixing asap :)

Reply:
No rush 👍 Thanks for working on that 😊